src/index.js

```diff
@@ -1,6 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter, Route } from 'react-router-dom';
 import './index.css';
 import App from './App';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
```

Comment:
So far, it looks great!
Can you create another file where you would put all our routes.
since putting all routes in inde.js is not best practice.